src/OmniSharp.MSBuild/ProjectManager.cs

```diff
@@ -193,31 +207,36 @@ private ProjectFileInfo LoadOrReloadProject(string projectFilePath, Func<(Projec
             _logger.LogInformation($"Loading project: {projectFilePath}");
 
             ProjectFileInfo projectFileInfo;
-            ImmutableArray<MSBuildDiagnostic> diagnostics;
 
             try
             {
+                ImmutableArray<MSBuildDiagnostic> diagnostics;
                 (projectFileInfo, diagnostics) = loadFunc();
 
-                if (projectFileInfo == null)
+                if (projectFileInfo != null)
+                {
+                    _logger.LogWarning($"Successfully loaded project file '{projectFilePath}'.");
+                }
+                else
                 {
                     _logger.LogWarning($"Failed to load project file '{projectFilePath}'.");
                 }
+
+                _eventEmitter.MSBuildProjectDiagnostics(projectFilePath, diagnostics);
             }
             catch (Exception ex)
             {
-                _logger.LogWarning($"Failed to load project file '{projectFilePath}'.", ex);
+                _logger.LogWarning($"Failed to load project file '{projectFilePath}' : {ex.ToString()}.");
```

Comment:
Why switch over to `ex.ToString()`? Passing `ex` should still log the exception.

Reply:
Also, it feels like this should probably be `LogError`.

Reply:
In my testing it didn't print the exception. But I will go ahead and revert this line to the original this is not related to the issue being fixed by these changes.

Reply:
Hmmm... we should check and see if the exception's not printing. I've definitely seen exceptions print in the past, but I wonder if it's not printing because this is a warning. I'll take a look.